templates/user/settings/profile.tmpl

```diff
@@ -73,7 +73,7 @@
 
 				<div class="field">
 					<div class="ui checkbox">
-						<label data-tooltip-content="{{ctx.Locale.Tr "settings.keep_email_private_popup"}}"><strong>{{ctx.Locale.Tr "settings.keep_email_private"}}</strong></label>
+						<label data-tooltip-content="{{ctx.Locale.Tr "settings.keep_email_private_popup" .SignedUser.GetPlaceholderEmail}}"><strong>{{ctx.Locale.Tr "settings.keep_email_private"}}</strong></label>
```

Comment:
"Use username@noreply.domain in commits to associate them with your account."

Does it really work this way? I am not sure it really works, this is a fake email address, I can't figure out how it could "associate"

Reply:
Here is the mapping of the noreply address to the account: https://github.com/go-gitea/gitea/blob/e544a277fe261aa5671a20751d6f83126579334a/models/user/user.go#L1132

Although it's ugly that it does not use the _GetPlaceholderEmail_, it works. I tried it in my commit for act runner:
https://gitea.com/hakito/act_runner/commit/f21bc49bd2d81be3ca47ca5b6210a4b05f46e073